### ISSUE TITLE
Illager banner in creative inventory places a normal white banner

### ISSUE THREAD

**Issue by dktapps** [pmmp/PocketMine-MP#2951]
### Issue description
Illager banner appears to use a non-standard method to define its pattern in NBT, which is not recognized by PM. This leads to the placement of a regular white banner when placing an illager banner.

This is considered a bug because the item appears in the creative inventory, but doesn't work as designed.

### Steps to reproduce the issue
1. Get an illager banner from the creative inventory.
2. Place it and notice it turns white.

### OS and versions

* PocketMine-MP: 3.8.3, master
* PHP: 7.3.5
* Server OS: linux
* Game version: all

**Comment by Wertzui123** [comment]
I noticed that too 🤔

**Comment by TheKnightGamerZ** [comment]
can't we not just add in src/pocketmine/tile/Banner.php PATTERN_ILLAGER and add the colors to it

**Comment by dktapps** [comment]
No, it's not that simple. Mojang specialized this one for reasons I'm not clear on.

**Comment by TheKnightGamerZ** [comment]
oh ok

**Comment by Wertzui123** [comment]
Okey.

**Comment by dktapps** [comment]
With the new serializer architecture in PM5, it should be possible to implement this as a separate item, avoiding the conditionally useless / broken pattern APIs for illager banners.

**Comment by dktapps** [comment]
Fixed by https://github.com/pmmp/PocketMine-MP/commit/8f9478e82fec72a8b9f75a90882a278af0ab3a08
